packages/grafana-ui/src/components/Table/AreaChartCell.tsx

```diff
@@ -0,0 +1,115 @@
+import { isArray } from 'lodash';
+import React, { FC } from 'react';
+
+import {
+  ArrayVector,
+  FieldType,
+  FieldConfig,
+  getMinMaxAndDelta,
+  FieldSparkline,
+  isDataFrame,
+  Field,
+} from '@grafana/data';
+import {
+  BarAlignment,
+  FieldColorModeId,
+  GraphDrawStyle,
+  GraphFieldConfig,
+  GraphGradientMode,
+  LineInterpolation,
+  TableAreaChartCellOptions,
+  TableCellDisplayMode,
+  VisibilityMode,
+} from '@grafana/schema';
+
+import { Sparkline } from '../Sparkline/Sparkline';
+
+import { TableCellProps } from './types';
+import { getCellOptions } from './utils';
+
+export const defaultAreaChartCellConfig: GraphFieldConfig = {
+  drawStyle: GraphDrawStyle.Line,
+  lineInterpolation: LineInterpolation.Smooth,
+  lineWidth: 1,
+  fillOpacity: 17,
+  gradientMode: GraphGradientMode.Hue,
+  pointSize: 1,
+  barAlignment: BarAlignment.Center,
+  showPoints: VisibilityMode.Never,
+};
+
+export const AreaChartCell: FC<TableCellProps> = (props) => {
+  const { field, innerWidth, tableStyles, cell, cellProps } = props;
+
+  const sparkline = getSparkline(cell.value);
+
+  if (!sparkline) {
+    return (
+      <div {...cellProps} className={tableStyles.cellContainer}>
+        no data
+      </div>
+    );
+  }
+
+  const range = getMinMaxAndDelta(sparkline.y);
+  sparkline.y.config.min = range.min;
+  sparkline.y.config.max = range.max;
+  sparkline.y.state = { range };
+
+  const cellOptions = getTableAreaChartCellOptions(field);
+
+  const config: FieldConfig<GraphFieldConfig> = {
+    color: cellOptions.color ? { mode: FieldColorModeId.Fixed, fixedColor: cellOptions.color } : field.config.color,
```

Comment:
think we should just use field.config.color here, no need to add a new custom field config option for it